### ISSUE TITLE
Mixpanel/Mixpanel-Swift.h file not found

### ISSUE THREAD

**Issue by younes200** [mixpanel/mixpanel-iphone#830]
**Integration Method:** CocoaPods
**Xcode Version:** 10.1
**Library Version:**  3.4.0
**Platform:** iOS
**Language:** Objective-C
**Description:**  'Mixpanel/Mixpanel-Swift.h' file not found in MPDisplayTrigger.m
**Expected Behavior:** 

The project target already have **EMBEDDED_CONTENT_CONTAINS_SWIFT** to `Yes` and I tried an clean up.
```
rm -rf ~/Library/Caches/CocoaPods
rm -rf Pods
rm -rf ~/Library/Developer/Xcode/DerivedData/*
pod deintegrate
pod setup
pod install
```

```
pod --version
1.6.1
```
I'm not using `use_frameworks!` in Podfile

Any idea ? :)
Thank you.

**Comment by Srijans-nd** [comment]
I'm facing the same issue. This happened when I upgraded to Swift 4.2 and updated MixPanel pod to 3.4.0

**Comment by younes200** [comment]
@Srijans-nd maybe for your case, you should use `pod 'Mixpanel-swift'` instead ;)

**Comment by zihejia** [comment]
Hi @younes200 , sorry about the inconvenience. Unfortunately for 3.4.0, you have to use `use_frameworks!`, if this is not what you want, you can use Carthage to integrate 3.4.0 or use 3.3.9 if you don't need the new in app notification feature. We will consider using pure Objective C in the next version.

**Comment by younes200** [comment]
I'll keep using 3.3.9 until next release without swift code. 
Thanks @zihejia.

**Comment by zihejia** [comment]
Hi @younes200 , we have released 3.4.1 and removed all swift related content. Again, thanks for using Mixpanel and bring this issue up.
